### DISCUSSION TITLE
Using functions of useStorage outside of plugin. How?

### DISCUSSION THREAD

**Discussion by ChusVa** [nuxt/nuxt#28342] [Questions]
I wrote plugin "_sever/plugins/storage.ts_":
```typescript
import fsLiteDriver from "unstorage/drivers/fs-lite"

const storage = useStorage()

export default defineNitroPlugin(() => {
    console.log('Load Plugin Storage...')
    storage.mount("/output", fsLiteDriver({ base: "./output" }))
    // storage.setItem("/output/test", "works")
})
```
My plugin is launches successfully. I want use functions _getItem()_ and _setItem()_ outside my plugin, but I don't know how. There is nothing written about this in the documentation. Are these features only available within the plugin? or is it somehow possible to use it outside the plugin?
The documentation is very unclear and contradictory.

**Comment by logotip4ik** [comment]
I suggest using nitro configuration to customize storage (https://nitro.unjs.io/guide/storage#configuration). Also, i think you don't need leading slash. One you configure the storage, you can do `const output = useStorage("output")`.  and `await output.setItem("test", {...})`. Every driver has it's own config (https://unstorage.unjs.io/), though all of them have mostly the same interface (whole point of unstorage). Unstorage is a thing that nuxt and nitro use to prove user easy kv storage
